web/console/src/app/components/Tokenomics/Spending/index.scss

```diff
@@ -1,25 +1,47 @@
 .spending {
 
     &__title {
-        font-size: 40px;
-        margin-bottom: 40px;
+        margin-bottom: 24px;
+        font-family: "Orbitron", sans-serif;
+        font-style: normal;
+        font-weight: 900;
```

Comment:
we can move this repeated code from child to parent (title, subtitle, description)